sdk/paloaltonetworks.ngfw/Azure.ResourceManager.PaloAltoNetworks.Ngfw/src/Generated/GlobalRulestackCertificateObjectCollection.cs

```diff
@@ -8,89 +8,101 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
-using System.Globalization;
+using System.Diagnostics;
 using System.Threading;
 using System.Threading.Tasks;
-using Autorest.CSharp.Core;
+using Azure;
 using Azure.Core;
 using Azure.Core.Pipeline;
+using Azure.ResourceManager;
 
 namespace Azure.ResourceManager.PaloAltoNetworks.Ngfw
 {
     /// <summary>
     /// A class representing a collection of <see cref="GlobalRulestackCertificateObjectResource"/> and their operations.
-    /// Each <see cref="GlobalRulestackCertificateObjectResource"/> in the collection will belong to the same instance of <see cref="GlobalRulestackResource"/>.
-    /// To get a <see cref="GlobalRulestackCertificateObjectCollection"/> instance call the GetGlobalRulestackCertificateObjects method from an instance of <see cref="GlobalRulestackResource"/>.
+    /// Each <see cref="GlobalRulestackCertificateObjectResource"/> in the collection will belong to the same instance of a parent resource (TODO: add parent resource information).
```

Comment:
haiyuan just fixed this issue.
Once this change takes effect, please regenerate, and this TODO should go away.